init-mongo.sh

```
@@ -0,0 +1,53 @@
+set -e
+
+# Handle cleanup on script exit
+cleanup() {
+    echo "Shutting down MongoDB..."
+    mongod --shutdown
+    exit 0
+}
+trap cleanup SIGTERM SIGINT
+
+mongod --replSet rs0 --bind_ip_all --dbpath /data/db &
+MONGOD_PID=$!
```

Shellcheck (0.10.0)

[warning] MONGOD_PID appears unused. Verify use (or export if used externally).

(SC2034)